packages/element-library/src/media/editPanMoveable.js

```diff
@@ -102,14 +104,7 @@ function EditPanMoveable({
       draggable
       throttleDrag={0}
       onDrag={({ dist }) => {
-        let [tx, ty] = dist;
-        if (flip?.vertical) {
-          ty = -ty;
-        }
-        if (flip?.horizontal) {
-          tx = -tx;
-        }
-        translateRef.current = [tx, ty];
```

Comment:
Note that the flip toggles was taking into consideration twice in this file - once in `onDrag` and again in `onDragEnd`. By making sure to apply the correctly transform from the start, this stuff here is not necessary.